uktrade/data-hub-frontend
# Make UK regions filter on contacts collection page a type ahead

## Description of change

This pull request updates the UK Regions filter in the contacts collection page to be a type ahead, rather than check boxes.

The original uk region functional test (in contacts) has been changed to test the new type ahead ability.

## Test instructions

- Go to the contacts collection page
- Filter UK Region by North West

_What should I see?_

## Screenshots
### Before

![Screenshot 2020-01-20 at 10 52 30](https://user-images.githubusercontent.com/22460823/72721014-076bbb00-3b73-11ea-94b8-6b37e66e0c3a.png)

### After

![Screenshot 2020-01-20 at 10 51 26](https://user-images.githubusercontent.com/22460823/72721027-0e92c900-3b73-11ea-81bf-7587b5fcc0c0.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
